NURESTConnection.js

```diff
@@ -157,6 +157,18 @@ export default class NURESTConnection extends NUObject {
         }
     }
 
+    static _parseStringToJSON = (string) => {
```

Comment:
rename string to something else for better readability and to avoid confusion with datatype string

Reply:
Done.